ui/components/component-library/text/text.scss

```diff
@@ -26,13 +26,11 @@ $text-variants: (
   }
 }
 
-
-
 .mm-text {
   // Set default styles
-  color: var(--color-text-default);
```

Comment:
Now that the specificity of the Box has been fixed. We should set the default by the utility prop so it this style doesn't override it